### DISCUSSION TITLE
2005/06/09/%E6%95%B4%E7%90%86%E7%A1%AC%E7%9B%98

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#36] [Announcements]
# 2005/06/09/%E6%95%B4%E7%90%86%E7%A1%AC%E7%9B%98

整理硬盘的时候发现了好多以前写的作业，共享给大家吧。

https://blog.qizhen.xyz/2005/06/09/%E6%95%B4%E7%90%86%E7%A1%AC%E7%9B%98

**Comment by ruanqizhen** [comment]
Dana
2005年06月20日 21:54 编辑
可否借我一阅？
